cmd/kube-controller-manager/app/plugins.go

```diff
@@ -53,3 +54,16 @@ func ProbeRecyclableVolumePlugins(flags VolumeConfigFlags) []volume.VolumePlugin
 	allPlugins = append(allPlugins, nfs.ProbeVolumePlugins(nfsConfig)...)
 	return allPlugins
 }
+
+// ProbeControllerPlugins collects all compiled-in plugins into an easy to use list.
+func ProbeControllerPlugins(flags ControllerConfigFlags) []controller.Plugin {
+	allPlugins := []controller.Plugin{}
+
+	_ = controller.ControllerConfig{
+		SyncPeriod: flags.ResourceQuotaSyncPeriod,
+	}
+
+	// allPlugins = append(allPlugins, resourcequota.ProbeControllerPlugins(config)...)
```

Comment:
This mimics how we added VolumeConfig.  This is here as an example of its usage and would eventually be replaced by real usage as controllers are converted into plugins.

Specific attributes are in the binary as flags, then they are passed generically via the config struct.